ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/subscription/request/SubscriptionRequestMapper.java

```diff
@@ -85,8 +85,8 @@ public UnsubscribeRequest mapUnsubscribeRequest(final JsonRpcRequest jsonRpcRequ
   }
 
   private WebSocketRpcRequest validateRequest(final JsonRpcRequest jsonRpcRequest) {
-    if (jsonRpcRequest instanceof WebSocketRpcRequest) {
-      return (WebSocketRpcRequest) jsonRpcRequest;
+    if (jsonRpcRequest.getRequestBody() instanceof WebSocketRpcRequest) {
```

Comment:
This reads a bit odd. Thinking should rename WebSocketRpcRequest to WebSocketRpcRequestBody as well

Reply:
done.